### ISSUE TITLE
Allow bundle traceability

### ISSUE THREAD

**Issue by guybedford** [systemjs/plugin-css#60]
The `document` code included for CSS injection makes tracing of the build file no longer possible. We should make sure this code will still run without errors in Node.

**Comment by guybedford** [comment]
Released in 0.1.15.

**Comment by HarelM** [comment]
Is this really fixed (it looks too old not to be but I have the same error)? I can't get systemjs builder to work and I'm getting this exact error:
```
[Error: Error on translate for first-file.js at file:///drive/folder/first-file.js
	document is not defined] originalErr: [ReferenceError: document is not defined] }
```
Is there a way to debug/understand where the error comes from?
I'm using version builder 0.16 and css-plugin 0.1.33.
Is there a tutorial I can use to debug the gulp task? I've wasted most of my day on this and didn't progressed an inch...
